File: src/Pug.Client/app/pug/pick-up-game.ts
import { Player } from "./player";
import { PugPhase } from "./phases/pug-phase";
import { Team } from "./team/team";

export interface PickUpGame {
    id: number;
    path: string[];
    pickablePlayers: Player[];
    currentPhase: PugPhase;
    teams: Team[];
}

File: Pug.ViewModels/PickUpGame.cs
using Pug.ViewModels.Phases;
using System.Collections.Generic;

namespace Pug.ViewModels
{
    public class PickUpGame
    {
        public string Id { get; set; }
        public IEnumerable<string> Path { get; set; }
        public BasePugPhase CurrentPhase { get; set; }
        public IEnumerable<Team> Teams { get; set; }
        public IEnumerable<Player> PickablePlayers { get; set; }
    }
}
